codespell_lib/data/dictionary.txt

```diff
@@ -52252,10 +52261,12 @@ skillfull->skillful, skilful, skillfully, skilfully,
 skillfullness->skillfulness, skilfulness,
 skipd->skipped
 skipe->skip, Skype,
+skipeable->skippable
```

Comment:
I hadn't added `skipable` because the [OED](https://www.oed.com/search/dictionary/?q=skipable) lists it as a variant of `skippable`.

But then, the Google Ngram Viewer shows a very clean winner in contemporary English:
https://books.google.com/ngrams/graph?content=skippable%2Cskipable&year_start=1800&year_end=2019&corpus=en-2019

So I have added it.